### ISSUE TITLE
NullPointerException in CustomSwipeRefreshLayout.onInterceptTouchEvent()

### ISSUE THREAD

**Issue by kassim** [xyxyLiu/SwipeRefreshLayout#10]
I'm getting the following crash reported by quite a few users. Any idea on the cause/solution? 

```
Fatal Exception: java.lang.NullPointerException: Attempt to invoke virtual method 'float android.view.MotionEvent.getY()' on a null object reference
       at com.reginald.swiperefresh.CustomSwipeRefreshLayout.onInterceptTouchEvent(CustomSwipeRefreshLayout.java:774)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2366)
       at com.reginald.swiperefresh.CustomSwipeRefreshLayout.dispatchTouchEvent(CustomSwipeRefreshLayout.java:741)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2839)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2514)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2839)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2514)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2839)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2514)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2839)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2514)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2839)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2514)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2839)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2514)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2839)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2514)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2839)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2514)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2839)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2514)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2839)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2514)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2839)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2514)
       at com.android.internal.policy.PhoneWindow$DecorView.superDispatchTouchEvent(PhoneWindow.java:2831)
       at com.android.internal.policy.PhoneWindow.superDispatchTouchEvent(PhoneWindow.java:1863)
       at android.app.Activity.dispatchTouchEvent(Activity.java:3046)
       at android.support.v7.view.WindowCallbackWrapper.dispatchTouchEvent(WindowCallbackWrapper.java:63)
       at android.support.v7.view.WindowCallbackWrapper.dispatchTouchEvent(WindowCallbackWrapper.java:63)
       at com.android.internal.policy.PhoneWindow$DecorView.dispatchTouchEvent(PhoneWindow.java:2792)
       at android.view.View.dispatchPointerEvent(View.java:10228)
       at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:5350)
       at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:5186)
       at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4626)
       at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4679)
       at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4645)
       at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:4787)
       at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4653)
       at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:4844)
       at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4626)
       at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4679)
       at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4645)
       at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4653)
       at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4626)
       at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:7312)
       at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:7190)
       at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:7151)
       at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:7422)
       at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:185)
       at android.view.InputEventReceiver.nativeConsumeBatchedInputEvents(InputEventReceiver.java)
       at android.view.InputEventReceiver.consumeBatchedInputEvents(InputEventReceiver.java:176)
       at android.view.ViewRootImpl.doConsumeBatchedInput(ViewRootImpl.java:7386)
       at android.view.ViewRootImpl$ConsumeBatchedInputRunnable.run(ViewRootImpl.java:7449)
       at android.view.Choreographer$CallbackRecord.run(Choreographer.java:920)
       at android.view.Choreographer.doCallbacks(Choreographer.java:695)
       at android.view.Choreographer.doFrame(Choreographer.java:625)
       at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:906)
       at android.os.Handler.handleCallback(Handler.java:739)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:158)
       at android.app.ActivityThread.main(ActivityThread.java:7229)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120)
```

**Comment by nrmtrifork** [comment]
Please accept PR #12 as it is an attempt to fix this bug

**Comment by nrmtrifork** [comment]
I closed PR #12 and created #13 instead. Now two nullpointer exceptions are checked for and eliminated. Please accept the PR @ReginaldLiu @xyxyLiu

**Comment by xyxyLiu** [comment]
fixed by #13
